src/menu_display.c

```diff
@@ -53,44 +53,51 @@ void display_rules(void)
     printf("    EXAMPLE: If the card on the top of the DISCARD pile is a blue 7, the player must put down a blue card OR 7 in any color.\n\n");
     //printf(" Alternatively, the player can put down a Wild card(See FUNCTIONS OF ACTION CARDS");
     printf("************************************************************************\n\n");
-   // printf(" Win the Game\n");
+    // printf(" Win the Game\n");
     printf("  Once a player has no cards left, the game is over. The player who exhausts all of his cards quickly is the winner of the game.\n");
     printf("  If no player is out of cards by the time the DRAW pile is depleted, the DISCARD pile is reshuffled and set as the draw pile. \n\n");
     printf("************************************************************************\n\n");
 
-     printf(" Inputs to play the game\n");
+    printf(" Inputs to play the game\n");
     printf("  To request a card from the Draw pile, Press N or n. \n");
     printf("  To end turn, Press E or e. \n");
     printf("  To discard a card - Say you want discard(RED,3), Press R3.\n");
```

Comment:
can you add " Say you want **to** discard" here?

Reply:
Updated.